config.yaml

```diff
@@ -13,7 +13,7 @@ options:
     default: true
   external_hostname:
     type: string
-    description: "External hostname this discourse instance should respond to. Defaults to the deployed application name."
```

Comment:
I'm not sure why we're changing this to include a `.local` prefix? We do that for indico because the application won't respond unless there's a "." in the hostname, but I don't think that's the case for discourse is it?

Reply:
Indeed. Let's do that